tests/test_appsettings.py

```diff
@@ -19,21 +22,30 @@ class SettingTestCase(SimpleTestCase):
     def _imported_object2():
         return "nothing"
 
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        fd, cls.temporary_file_name = mkstemp()
```

Comment:
`mkstemp` opens the file as well. Should we not at least close it after the creation?

Reply:
* I suggest using `NamedTemporaryFile` instead. It deletes the file after usage.
* To reduce a possibility of a test interference, I'd use different file for each test, i.e. move this into a `setUp`.

Reply:
I disagree with the `setUp`. Majority of tests doesn't need the temp file at all. If anything, we could transfer this operation to the tests themselves, or separate them into another `TestCase`.

Reply:
+1 for separating the tests.

Reply:
Operation transferred to tests themselves, tests separated to a special VIP class.